resources/lib/WonderfulSubsBrowser.py

```diff
@@ -62,14 +64,23 @@ def _parse_filtered_anime_view(self, res, base):
                                               "%s/%s" % (base["url"], self._FILTER_FLAVOR[3:6]),
                                               True,
                                               base["image"],
-                                              base["plot"]))
+                                              base["plot"],
+                                              self._draw_cm))
 
         return result
 
     def _parse_history_view(self, res):
         name = res
         return utils.allocate_item(name, "search/" + name + "/1", True)
 
+    def _draw_cm(self, sysaddon, name):
```

Comment:
this doesnt belong to WonderfulSubs browser
it should be part of default.py and passed as callback to the others.